.prettierrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  semi: true,
+  singleQuote: true,
+  trailingComma: 'es5',
+  printWidth: 120,
+  overrides: [
+    {
+      files: '*.md',
+      options: {
+        printWidth: 80,
+      },
+    },
+  ],
```

Comment:
I have found it to be easier to read markdown files if they are formatted to 80 width.
This don't affect the text rendering but keeps code blocks from getting too wide.

Reply:
That might be ok however I am wondering if this is a good experience for editing 

https://github.com/electricitymap/electricitymap-contrib/edit/master/DATA_SOURCES.md

which is a file often edited by non-technical contributors, likely editing directly in the github web editor

Reply:
I ran prettier on `DATA_SOURCES.md` locally and it don't change any of the text. The only changes where to whitespace, quotes and some html codes (like turning `&#32;` into spaces in link tooltips).